css-pseudo-classes/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>CSS Pseudo Classes</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <h1>Hoover Button</h1>
+    <form>
+        <input type="button" class="button" value="I'm a button!">
```

Comment:
This is fine, just important to point out that Buttons don't always have to be in forms! They can be used for any type of click functionality outside of forms as well.